package.json

```diff
@@ -39,7 +39,8 @@
   "activationEvents": [
     "onCommand:extension.fabric8AnalyticsWidgetFullStack",
     "workspaceContains:**/package.json",
-    "workspaceContains:**/pom.xml"
+    "workspaceContains:**/pom.xml",
+    "workspaceContains:**/requirements.txt"
```

Comment:
why are we this activating are we supporting python.

Reply:
yes it would be there :)